ami/models/components/forward_dynamics.py

```diff
@@ -0,0 +1,31 @@
+import torch
+import torch.nn as nn
+from torch import Tensor
+from torch.distributions import Normal
+
+from .stacked_hidden_state import StackedHiddenState
+
+
+class ForwardDynamics(nn.Module):
+    def __init__(
+        self,
+        observation_flatten: nn.Module,
+        action_flatten: nn.Module,
+        obs_action_projection: nn.Module,
+        core_model: StackedHiddenState,
+        obs_hat_dist_head: nn.Module,
+    ) -> None:
+        super().__init__()
+        self.observation_flatten = observation_flatten
+        self.action_flatten = action_flatten
+        self.obs_action_projection = obs_action_projection
+        self.core_model = core_model
+        self.obs_hat_dist_head = obs_hat_dist_head
+
+    def forward(self, obs: Tensor, hidden: Tensor, action: Tensor) -> tuple[Normal, Tensor]:
```

Comment:
出力型は`Normal`ではなく、 `Distribution`にしてください。
```suggestion
    def forward(self, obs: Tensor, hidden: Tensor, action: Tensor) -> tuple[Distribution, Tensor]:
```

Reply:
修正しました https://github.com/MLShukai/ami/pull/121/commits/2765c65c9d7803d5c0bd332ac06af8f2350b2a49